go.sum

```diff
@@ -63,5 +65,7 @@ google.golang.org/genproto v0.0.0-20180716172848-2731d4fa720b/go.mod h1:JiN7NxoA
 google.golang.org/grpc v0.0.0-20180510165436-d07538b1475e h1:X6P5OjvrKe7dZIhFpqU7P6JWZVwnSvT8bB0iRauPo3I=
 google.golang.org/grpc v0.0.0-20180510165436-d07538b1475e/go.mod h1:yo6s7OP7yaDglbqo1J04qKzAhqBH6lvTonzMVmEdcZw=
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
+gopkg.in/yaml.v2 v2.2.1-0.20180326061214-4fc5987536ef h1:KngkwkexysX8F+cPblZfFFu3eAF/xsJJej9Jt93h3qI=
+gopkg.in/yaml.v2 v2.2.1-0.20180326061214-4fc5987536ef/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
 gopkg.in/yaml.v2 v2.2.1 h1:mUhvW9EsL+naU5Q3cakzfE91YhliOondGd6ZrsDBHQE=
```

Comment:
Remove the old lines: `rm -f go.sum; go test ./kythe/...`

Reply:
done, thanks